responder/models.py

```
@@ -249,7 +248,7 @@ def accepts(self, content_type):
         """Returns ``True`` if the incoming Request accepts the given ``content_type``."""
         return content_type in self.headers.get("Accept", [])
 
-    async def media(self, format: t.Union[str, t.Callable] = None):  # noqa: A002
+    async def media(self, format: str | Callable = None):  # noqa: A002
```

Ruff (0.15.6)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)